### PR TITLE
Pairs in c++

### PR DESCRIPTION
**Pull request by geetika005**
<hr>

## Description 📜

Pairs in C++ standard template library #2257
There is basic introduction to pair.

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Review by vasu-1 — CHANGES_REQUESTED** [review]
- please put your content inside C/CPP's respective subdirectories.
- for this move your pair.md file inside ` /C_CPP/Data Structures/Standard Template Library/ `

**Comment by geetika005** [comment]
ok i will do that

**Comment by vasu-1** [comment]
Note : You can only put your content in C/CPP's Subdirectories.

**Comment by geetika005** [comment]
Is it ok now

**Review by vasu-1 — CHANGES_REQUESTED** [review]
Hey @geetika005 i have done some markdown formatting.
kindly see , learn & make those changes.
[link](https://github.com/vasu-1/vasu-1/blob/main/pair.txt)
see this link...

**Comment by vasu-1** [comment]
Also, add a link of your code in ` Standard Template Library/readme.md `.

- add this line in `Standard Template Library/readme.md`
` - [Pair](./pair.md) `

**Comment by geetika005** [comment]
Done with the changes ,please check it now.

**Review by vasu-1 — APPROVED** [review]
Great @geetika005 ~!!!!!

Good to go for Merge @Suvraneel

**Comment by Suvraneel** [comment]
@vasu-1  Excellent reviewing ❤️ 

@geetika005  Perfect & neat explanation... Would love if it was atleast a 2 minute read
Try adding make_pair & tie() etc...

File: C_CPP/Data Structures/Standard Template Library/pair.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+### Swap Functions
+
+  - This function is used to swap the content of two pair of same type with each other.  
+ 
+ ### Syntax 
+ ```c++
+  PAIR1.swap(PAIR2); 
+  ```
+
```
Comment (Suvraneel) [inline C_CPP/Data Structures/Standard Template Library/pair.md:59]: make pair
tie () etc....

Amplify & Enhance doc

**Review by Suvraneel — CHANGES_REQUESTED** [review]

**Comment by geetika005** [comment]
Made the changes sir.

File: C_CPP/Data Structures/Standard Template Library/pair.md (hunk 1 above)
Comment (geetika005) [reply C_CPP/Data Structures/Standard Template Library/pair.md:80]: added

**Review by Suvraneel — APPROVED** [review]
Awesome ❤️ documentation :octocat: 
Looking forwad to more great contribs from you 💯

**Pull request merged**